### ISSUE TITLE
Loss = nan

### ISSUE THREAD

**Issue by kinhunt** [ardiya/HashtagPrediction#6]
```
INFO:tensorflow:global step 2454: loss = 0.6953 (1.61 sec/step)
INFO:tensorflow:global step 2455: loss = 0.6953 (1.61 sec/step)
INFO:tensorflow:global step 2456: loss = 0.6953 (1.61 sec/step)
INFO:tensorflow:global step 2457: loss = 0.6953 (1.61 sec/step)
INFO:tensorflow:global step 2458: loss = nan (2.34 sec/step)
INFO:tensorflow:global step 2459: loss = nan (1.50 sec/step)
INFO:tensorflow:global step 2460: loss = nan (1.61 sec/step)
INFO:tensorflow:global step 2461: loss = nan (1.59 sec/step)
INFO:tensorflow:global step 2462: loss = nan (1.59 sec/step)
INFO:tensorflow:global step 2463: loss = nan (1.61 sec/step)
```
